<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <test thread-count="5" name="Submit Order(failed)">
    <classes>
      <class name="rivekajoshi.Test.ErrorValidationTest">
        <methods>
          <include name="teardown"/>
          <include name="launch"/>
          <include name="ProductErrorValidation"/>
          <include name="submittingorder"/>
        </methods>
      </class> <!-- rivekajoshi.Test.ErrorValidationTest -->
      <class name="rivekajoshi.Test.SubmitOrder">
        <methods>
          <include name="teardown"/>
          <include name="launch"/>
          <include name="submittingorder"/>
        </methods>
      </class> <!-- rivekajoshi.Test.SubmitOrder -->
    </classes>
  </test> <!-- Submit Order(failed) -->
  <test thread-count="5" name="Error ValidationTest(failed)">
    <classes>
      <class name="rivekajoshi.Test.ErrorValidationTest">
        <methods>
          <include name="teardown"/>
          <include name="launch"/>
          <include name="ProductErrorValidation"/>
          <include name="submittingorder"/>
        </methods>
      </class> <!-- rivekajoshi.Test.ErrorValidationTest -->
    </classes>
  </test> <!-- Error ValidationTest(failed) -->
</suite> <!-- Failed suite [Suite] -->
